static/locales/ur.yaml

```diff
@@ -0,0 +1,790 @@
+    # $ is replaced with the default custom arguments for the current player, if defined.
+    DefaultCustomArgumentsTemplate: '(پہلے سے طے شدہ: ''$'')'
+  Subscription Settings:
+    Fetch Feeds from RSS: 'فعال ہونے پر، FreeTube اپنے ڈیفالٹ کے بجائے RSS کا استعمال کرے گا۔
+       آپ کی سبسکرپشن فیڈ حاصل کرنے کا طریقہ۔ آر ایس ایس تیز ہے اور آئی پی کو بلاک کرنے سے روکتا ہے،
+       لیکن کچھ معلومات فراہم نہیں کرتا ہے جیسے ویڈیو کا دورانیہ یا لائیو اسٹیٹس'
+  Privacy Settings:
+    Remove Video Meta Files: 'فعال ہونے پر، FreeTube ویڈیو پلے بیک کے دوران بنائی گئی میٹا فائلوں کو خود بخود حذف کر دیتا ہے،
+       جب دیکھنے کا صفحہ بند ہوتا ہے۔'
+
+# Toast Messages
+Local API Error (Click to copy): ''
+Invidious API Error (Click to copy): ''
+Falling back to Invidious API: ''
+Falling back to the local API: ''
+This video is unavailable because of missing formats. This can happen due to country unavailability.: 'یہ
+   ویڈیو غائب فارمیٹس کی وجہ سے دستیاب نہیں ہے۔ یہ ملک کی وجہ سے ہوسکتا ہے۔
+   غیر دستیابی'
+Subscriptions have not yet been implemented: 'سبسکرپشنز کو ابھی تک نافذ نہیں کیا گیا ہے۔'
+Unknown YouTube url type, cannot be opened in app: 'نامعلوم YouTube url قسم، ایپ میں نہیں کھولی جا سکتی'
+Hashtags have not yet been implemented, try again later: 'ہیش ٹیگز ابھی لاگو نہیں ہوئے، بعد میں دوبارہ کوشش کریں۔'
+Loop is now disabled: 'لوپ اب غیر فعال ہے۔'
+Loop is now enabled: 'لوپ اب فعال ہے۔'
+Shuffle is now disabled: 'شفل اب غیر فعال ہے۔'
+Shuffle is now enabled: 'شفل اب فعال ہے۔'
+The playlist has been reversed: 'پلے لسٹ کو الٹ دیا گیا ہے۔'
+Playing Next Video: 'اگلا ویڈیو چل رہا ہے۔'
+Playing Previous Video: 'پچھلا ویڈیو چل رہا ہے۔'
+Playing Next Video Interval: 'اگلی ویڈیو بغیر کسی وقت چلائی جا رہی ہے۔ منسوخ کرنے کے لیے کلک کریں۔ | اگلی ویڈیو {nextVideoInterval} سیکنڈ میں چلائی جا رہی ہے۔ منسوخ کرنے کے لیے کلک کریں۔ | اگلی ویڈیو {nextVideoInterval} سیکنڈ میں چلائی جا رہی ہے۔ منسوخ کرنے کے لیے کلک کریں۔'
+Canceled next video autoplay: 'اگلی ویڈیو آٹو پلے منسوخ کر دی گئی۔'
+# $ is replaced with the default Invidious instance
+Default Invidious instance has been set to $: 'ڈیفالٹ Invidious مثال کو $ پر سیٹ کر دیا گیا ہے۔'
+Default Invidious instance has been cleared: 'ڈیفالٹ Invidious مثال کو صاف کر دیا گیا ہے۔'
+'The playlist has ended.  Enable loop to continue playing': 'پلے لسٹ ختم ہو گئی ہے۔ فعال
+   کھیل جاری رکھنے کے لیے لوپ'
+Age Restricted:
+  # $contentType is replaced with video or channel
+  This $contentType is age restricted: 'یہ $ عمر کی پابندی ہے۔'
+  Type:
+    Channel: 'چینل'
+    Video: 'ویڈیو'
+External link opening has been disabled in the general settings: 'عام ترتیبات میں بیرونی لنک کھولنے کو غیر فعال کر دیا گیا ہے۔'
+Downloading has completed: '"$" نے ڈاؤن لوڈ مکمل کر لیا ہے۔'
+Starting download: '"$" کا ڈاؤن لوڈ شروع ہو رہا ہے'
+Downloading failed: '"$" کو ڈاؤن لوڈ کرنے میں ایک مسئلہ تھا'
+Screenshot Success: 'اسکرین شاٹ کو بطور "$" محفوظ کیا گیا'
+Screenshot Error: 'اسکرین شاٹ ناکام ہو گیا۔ $'
+
+Yes: Yes
+No: No
```

Comment:
```suggestion
No: ''
```

Reply:
Sorry, its done, can you please tell me how can I be running the project locally?  will `npm start` or `yarn dev` will work?

Reply:
`yarn install` to install the dependencies and then `yarn dev` to run in dev mode.

Reply:
What absidue said + if you're trying to test out your language, you'll need to add "ur" to https://github.com/FreeTubeApp/FreeTube/blob/cb63b12caf962ab7ed2a1d7b27774acd4cf6c3ab/src/renderer/i18n/index.js#L11